libs/net/dal/Migrations/1.0.110/Up/PostUp/00-Settings.sql

```diff
@@ -0,0 +1,21 @@
+INSERT INTO public."setting" (
```

Comment:
Make sure the default report template is pointing to the correct value.  This however is linked to the friendly name of the template, which could be manually changed...